runtime/commonMain/src/kotlinx/serialization/modules/SerialModuleExtensions.kt

```diff
@@ -22,12 +22,23 @@ public fun <T : Any> SerialModule.getContextual(value: T): KSerializer<T>? {
     return getContextual(value::class)?.cast()
 }
 
+/**
+ * Attempts to retrieve a serializer from the current module and, if not found, fallbacks to [serializer] method
+ */
+@OptIn(UnsafeSerializationApi::class)
```

Comment:
Shouldn't these 3 functions be propagating Unsafe?

Reply:
The first one should not, because it is typeOf-based.

The second and third are deprecated, so no sense in adding it there